app/components/application/FeedbackCard/FeedbackCard.tsx

```diff
@@ -13,36 +14,55 @@ export default function FeedbackBox({ className }: { className?: string }) {
   const {
     t: { feedback },
   } = useTranslate()
+  const [hovered, setHovered] = useState<boolean>(false)
+
+  const handleMouseEnter = () => {
+    setHovered(true)
+  }
+
+  const handleMouseLeave = () => {
+    setHovered(false)
+  }
 
   return (
-    <Card>
-      <div className="feedback box">
-        <div className="top">
-          <div className="row">
-            <div className="spaceholder">
-              <img
-                alt={feedback.feedbackShareAltText}
-                aria-hidden="true"
-                className="image"
-                src="/share-feedback.svg"
-              />
-            </div>
-            <div>
-              <h3 className="title">{feedback.feedbackTitle}</h3>
-              <p className="bodytext">{feedback.feedbackSubText}</p>
+    <a
+      aria-label="Click to share feedback with the Pokt Team in discord"
```

Comment:
Should this come from `useTranslate`?

Reply:
Probably, if we wanna go in that direction its fine with me, but we don't really support i18n and I can't be bothered most of the time since we're not actually supporting it. Should probably make a mental note to update other parts of the code to use the translate.

Reply:
That also raises the question if its even worth it to keep this around, like do we actually have plans to support multiple languages.

Reply:
I think the discussion here is about a bigger scope and if we need to make a decision about this sooner or later we can keep it as is but maybe add a TODO or a note as you mentioned to take this later with everyone and make a decision about it.